src/engraving/tests/barline_data/barlinedelete-ref.mscx

```diff
@@ -5,7 +5,7 @@
     <Style>
       <lastSystemFillLimit>0</lastSystemFillLimit>
       <defaultsVersion>301</defaultsVersion>
-      <Spatium>1.76389</Spatium>
+      <spatium>1.76389</Spatium>
```

Comment:
Hm, now the close tag still needs to be adjusted...

Reply:
OK, will fix